src/include/migraphx/stringutils.hpp

```diff
@@ -31,6 +31,7 @@
 #include <unordered_map>
 #include <vector>
 #include <migraphx/config.hpp>
+#include <migraphx/tensor_view.hpp>
```

Comment:
I do not find appropriate place. Do you have any suggestion ? 
Otherwise i can do either of the following :
1. I'll have to create another header file just for the `as_number`. 
2. I can duplicate the function in both places. It is simple enough.

Reply:
Either one is fine. if you do "2" make sure the name doesnt conflict(perhaps it can be put into a seperate namespace).

Reply:
Added separate header file